aries/messages/src/msg_fields/protocols/did_exchange/request.rs

```diff
@@ -52,7 +52,7 @@ mod tests {
         msg_types::protocols::did_exchange::DidExchangeTypeV1_0,
     };
 
-    fn request_content() -> RequestContent {
+    pub fn request_content() -> RequestContent {
```

Comment:
I would much rather move the test into this module instead of exposing random test utilities from wherever. This practice leads to nothing but headache during test refactoring.

Reply:
The reason why I placed it here is because I am not something did-exchange specific, but rather macro generated on `MsgParts`, and just happened to choose did-exchange as one of the specific MsgParts concrete type for it. 
I can move it if you insist, but I'd argue this will bury the test a level lower where it actually belongs.

Reply:
I understand the reason for placing it where it is but I would still move it. It can be seen as testing a property or behavior of the DID exchange `Request` message.

Reply:
moved